app/assets/stylesheets/application.css

```diff
@@ -14,3 +14,8 @@
  *= require_self
  */
 
+*{
```

Comment:
This is too powerful I feel. Do you need this for all the elements ??

Reply:
I think I can take it off. I will resolve it soon.